wdl/transforms/new-base/src/main/scala/wdl/transforms/base/ast2wdlom/AstNodeToMetaKvPair.scala

```diff
@@ -37,6 +37,7 @@ object AstNodeToMetaKvPair {
           case ("integer", i) => Try(MetaValueElementInteger(i.toInt)).toErrorOr
           case ("float", f) => Try(MetaValueElementFloat(f.toDouble)).toErrorOr
           case ("boolean", b) => Try(MetaValueElementBoolean(b.toBoolean)).toErrorOr
+          case ("string", s) => Try(MetaValueElementString(s)).toErrorOr
```

Comment:
This looks... correct... but unrelated to the change to allow empty call blocks?

Reply:
With the newly-generated parser, this change is required for the `meta` test cases in [Ohad's recent PR](https://github.com/broadinstitute/cromwell/pull/4729) to pass.

The cases must have worked before because they fell under the `"StaticString"` case.